sycl/include/sycl/detail/pi.def

```diff
@@ -150,6 +150,10 @@ _PI_API(piextCommandBufferNDRangeKernel)
 _PI_API(piextCommandBufferMemcpyUSM)
 _PI_API(piextCommandBufferMemBufferCopy)
 _PI_API(piextCommandBufferMemBufferCopyRect)
+_PI_API(piextCommandBufferMemBufferWrite)
+_PI_API(piextCommandBufferMemBufferWriteRect)
+_PI_API(piextCommandBufferMemBufferRead)
+_PI_API(piextCommandBufferMemBufferReadRect)
```

Comment:
We might need to update the SYCL ABI tests to reflect these, see the `sycl/test/abi/` changes in https://github.com/intel/llvm/pull/9992

Should be able to run these tests locally by pointing lit to them `./bin/llvm-lit --param sycl_be=ext_oneapi_level_zero --param target_devices=gpu -sv <repo root>/sycl/test/abi/`

Reply:
The tests are successful except abi/layout_handler.cpp
But the failure is not related to this PR (also existing in the sycl-graph-develop branch)

Reply:
cool thanks 🙂 The layout tests are hard to keep up-to-date with upstream anyway, even if we fix them now we might have to update them later. Happy to avoid the duplicated effort and leave them for this PR and only add the extra symbols where needed, as you have done.